compiler/modules/types/listProj.bal

```diff
@@ -52,31 +52,28 @@ function listProjPath(Context cx, int k, Conjunction? pos, Conjunction? neg) ret
                 Atom d = p.atom;
                 p = p.next; 
                 lt = cx.listAtomType(d);
-                int newLen = int:max(members.length(), lt.members.length());
-                if members.length() < newLen {
+                int newLen = int:max(members.fixedLength, lt.members.fixedLength);
```

Comment:
Can we use listIntersectWith here?

Reply:
oh yes, I'll replace these with `listIntersectWith`